src/scripts/browserDeviceProfile.js

```diff
@@ -1259,17 +1263,16 @@ export default function (options) {
     });
 
     if (browser.web0s && supportsDolbyVision(options)) {
-        // Disallow direct playing of DOVI media in containers not mp4.
-        // This paired with the "Prefer fMP4-HLS Container" client playback setting enables DOVI playback on webOS.
+        // Disallow direct playing of DOVI media in containers not ts or mp4.
         profile.CodecProfiles.push({
             Type: 'Video',
-            Container: '-mp4',
+            Container: '-ts,mp4',
```

Comment:
The order doesn't matter, but I think we should stick to alphabetical order.
```suggestion
            Container: '-mp4,ts',
```